### DISCUSSION TITLE
GitHub Projects: Insights query have been returning 503 recently

### DISCUSSION THREAD

**Discussion by kallehauge** [community/community#47240] [Projects and Issues]
### Select Topic Area

Bug

### Body

We have a specific chart that recently started returning **503** responses when fetching data through `https://github.com/memexes/{projectID}/insights/query`. The error response is so simplistic that I cannot debug it further myself:

```json
{"message":"Error fetching insights data"}
```

## Request Id examples

* F939:90FC:8E95595:93D6916:63EA3378
* F939:90FC:8E5A240:939A1C4:63EA32E1
* F939:90FC:8EB7275:93F90E1:63EA33CE
* F939:90FC:8EC0A7A:9402C52:63EA33E6

## Insight chart configuration

A line chart that sum the value of a `Number` field (named `Estimate` in the screenshot below) and groups the results by the assignees on GitHub Issues.

![Screenshot 2023-02-13 at 14 04 25](https://user-images.githubusercontent.com/3846700/218465941-43300630-e63d-427a-ac71-ccc0b1abd1f4.jpg)

## Previous discussions

It's a bit tricky trying to find "Projects" specific issues since not all of them are tagged with the `Projects` label, but I cannot find any other discussions about 503:

* https://github.com/orgs/community/discussions?discussions_q=label%3AProjects+503
* https://github.com/orgs/community/discussions?discussions_q=503+insight

**Comment by chiedo** [comment]
Thanks for reporting this @kallehauge and sorry you're running into this. I'll keep an eye out for this and see if I can reproduce it.

**Reply by kallehauge** [reply]
@chiedo I can't seem to find anyone else experiencing it after ~1 month (although I still find it difficult to filter out issues specifically for `Projects`, so I might have missed something 😅).

Have you had any luck reproducing it?
AFAIK then it only seems to happen when grouping by assignees. All other grouping options seems to be running smoothly (I've specifically tried grouping successfully with: labels, milestones, custom fields, repository, iterations)

**Reply by chiedo** [reply]
Not yet! 🙃 But I've shared the issue with colleagues to see if anyone else has.
